webapp/src/tutorial.tsx

```diff
@@ -433,12 +435,22 @@ export class TutorialCard extends data.Component<TutorialCardProps, TutorialCard
         }
     }
 
+    private handleResize() {
+        const options = this.props.parent.state.tutorialOptions;
+        if (this.resizeDebounceTimer) {
+            clearTimeout(this.resizeDebounceTimer);
+        }
+        this.resizeDebounceTimer = setTimeout(()=>(this.setShowSeeMore(options.autoexpandStep)), 500);
```

Comment:
we have a helper to do debouncing https://github.com/microsoft/pxt/blob/d57cb0b70b95b552c26e8e8cf8d8685d2b7b1c82/pxtlib/util.ts#L470

Reply:
ooooooooooo great I'll use that! tytytyty